server/views/formPages/incident/relocationAndInjuries.html

```diff
@@ -30,19 +30,43 @@ <h1 class="govuk-heading-xl mainHeading">{{ pageTitle }}</h1>
 
 
 
-    {{ incidentMacro.radio( {
-      text: "Was the prisoner compliant?",
-      name: "relocationCompliancy",
-      value: data.relocationCompliancy,
-      options: [{
-        value: true,
-        label: "Yes"
-      }, {
-        value: false,
-        label: "No"
-      }],
-      errorMessage: errors | findError('relocationCompliancy')
-    })}}
+    {{ incidentMacro.radiosWithNestedRadios({
+      primaryQuestion: {
+        text: "Was the prisoner compliant?",
+        name: "prisonerCompliant",
+        value: data.prisonerCompliant,
+        options: [{
+          value: true,
+          label: "Yes"
+        }, {
+          value: false,
+          label: "No"
+        }],
+        errorMessage: errors | findError('prisonerCompliant')
+
+      },
+      followUpQuestion: {
+        text: "What was the type of relocation?",
+        name: "relocationType",
+        value: data.relocationType,
+        errorMessage: errors | findError('relocationType'),
+        options: [{
```

Comment:
We tend to store enumerated types in here: `/Users/andrewlee/code/use-of-force/server/config/types.js`.

you can then use the `toOptions` helper, e.g:

```    
   options: data.types.ControlAndRestraintPosition | list | extractAttr('value') | toOptions('value', 'label')
 ```
the idea being that we have encoded the 'Type' of the data explicitly rather than having the type implicitly defined in the html.

Reply:
I tried to use the same pattern as the other components on the page. 
Have now changed to using types

Reply:
done